src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,257 @@
+1. Use the token in place of a password in your database connection string.
 
-## Provision a service account
+{% include_cached copy-clipboard.html %}
+~~~shell
+cockroach sql --url "postgresql://{SQL_USERNAME}:{JWT_TOKEN}@{CLUSTER_HOST}:26257?options=--crdb:jwt_auth_enabled=true" --certs-dir={CLUSTER_CERT_DIR}
+~~~
 
-For access to a CockroachDB cluster, an identity in an external IdP must correspond to a SQL user on the target cluster, so creating a service for your cluster entails two steps:
+~~~txt
+Welcome to the cockroach SQL interface...
+~~~
 
-- Create a service account/IAM username with your external IdP (for example, GCP).
-- [Create a SQL user]({% link {{ page.version.version }}/create-user.md %}) in your cluster.
+## Configure your cluster for SSO
 
-The **correspondence** between IdP identity and SQL username is crucial. This determined by [your cluster's identity mapping](#configure-your-clusters-identity-mapping).
+**Prerequisites**:
 
-## Configure your cluster to accept your external identity provider
+You must have the ability to update your cluster settings, which requires a SQL user either the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or the `MODIFYCLUSTERSETTING` role option.
 
-In order to authenticate a service account to a CockroachDB {{ site.data.products.cloud }} cluster using a JWT issuer, you must update several cluster settings in the `server.jwt_authentication` namespace, as well as the [`identity_map.configuration`](#configure-your-clusters-identity-mapping)
 
-- [`enabled`](#server-jwt_authentication-enabled): JWT token authentication must be allowed on your cluster.
-- [`jwks`](#server-jwt_authentication-jwks): A list of public signing keys for allowed IdPs. You'll need to add your IdP's key.
-- [`issuers`](#server-jwt_authentication-issuers): A list of accepted token issuers. You'll need to add your IdP.
-- [`audience`](#server-jwt_authentication-audience): A list of audiences (or targets) for authentication, most relevantly, clusters.
+### Cluster Settings
 
-{{site.data.alerts.callout_success}}
-The required information for a given IdP is published on that IdP's `.well-known/openid-configuration` path (for example, `https://cockroachlabs.cloud/.well-known/openid-configuration` for CockroachDB {{ site.data.products.cloud }} or `https://accounts.google.com/.well-known/openid-configuration` for GCP.
-{{site.data.alerts.end}}
+| Cluster Setting | Description 
+|-----------------|------ 
+| `server.jwt_authentication.enabled` | Defaults to `false`, must be set to `true` to enable embedded JWT generation.
+| `server.jwt_authentication.jwks` | A list of public signing keys for allowed IdPs; must include your IdP's key. 
+| `server.jwt_authentication.issuers` | A list of accepted token issuers; must include your IdP.
+| `server.jwt_authentication.audience` | A list of audiences (or targets) for authentication, most relevantly, clusters.
```

Comment:
This sentence is a little weird. It's the audience that refers to this cluster. So tokens with that audience will be accepted.